content/docs/1.28/operator.md

```diff
@@ -58,23 +58,27 @@ The following instructions will create the `observability` namespace and install
 Make sure your `kubectl` command is properly configured to talk to a valid Kubernetes cluster. If you don't have a cluster, you can create one locally using [`minikube`](https://kubernetes.io/docs/tasks/tools/install-minikube/).
 {{< /info >}}
 
+{{< info >}}
+Because the master branch could change, we are using pined versions now, if you are using master branch please make sure do the proper changes
```

Comment:
Let's remove this. It looks a bit odd on the website. 

We have already broken people that were using master, no reason to mention it here.